accounts/validation.py

```diff
@@ -60,7 +61,21 @@ def validate_password_field(password, confirm_password):
             status=status.HTTP_400_BAD_REQUEST,
         )
 
-    if not re.search(r"[!@#$%^&*()_+]", password):
+    character_pattern = r"^(?=.*[A-Z])(?=.*[a-z]).+$"
+    if not regex.search(character_pattern, password):
+        return Response(
+            {"error": "Password must contain at least one small or capital letter."},
```

Comment:
@usanaphtal112  Please recheck this error message again ("error": "Password must contain at least one small or capital letter."). I think it supposed to be "...at least one lower case and upper case letter. ( Not **"or"**)

![request](https://github.com/SahanaTeam/sahaget-bn/assets/103914775/8fa046dd-10b9-455a-aa9e-2731ebe7efdb)

![response](https://github.com/SahanaTeam/sahaget-bn/assets/103914775/0feb43dd-2ce8-41ac-bbea-2a8c96018d75)